armi/reactor/tests/test_components.py

```diff
@@ -217,13 +217,21 @@ def test_solid_material(self):
         .. test:: Determine if material is solid.
             :id: T_ARMI_COMP_SOLID
             :tests: R_ARMI_COMP_SOLID
+
+        .. test:: Components have material properties.
+            :id: T_ARMI_COMP_MAT
+            :tests: R_ARMI_COMP_MAT
         """
+        self.assertIn("HT9", str(self.component.getProperties()))
+
         self.component.material = air.Air()
         self.assertFalse(self.component.containsSolidMaterial())
 
         self.component.material = alloy200.Alloy200()
         self.assertTrue(self.component.containsSolidMaterial())
 
+        self.assertIn("Alloy200", str(self.component.getProperties()))
+
```

Comment:
I think we probably need to take this a little further and show that there are actually properties here. I don't immediately see how testing for the material name is proving this.

Reply:
```suggestion

        .. test:: Components have material properties.
            :id: T_ARMI_COMP_MAT
            :tests: R_ARMI_COMP_MAT
        """
        self.assertTrue(hasattr(self.component.material, "density"))

        self.component.material = air.Air()
        self.assertFalse(self.component.containsSolidMaterial())

        self.component.material = alloy200.Alloy200()
        self.assertTrue(self.component.containsSolidMaterial())

        self.assertTrue(hasattr(self.component.material, "density"))

```

What do you think of that instead? I deleted the old asserts, but we could leave those in as well.

Reply:
Okay, I added those. And kept the old ones.